basic-javascript/project-1.spec.js

```diff
@@ -1,3 +1,247 @@
 const helpers = require('./project-1');
 
-// start testing!
+//========== MULTIPLY BY 10
+describe('multiplyByTen()', () => {
+    it('should multiply provided number by ten', () => {
+        const expected = 50;
+        const actual = helpers.multiplyByTen(5);
+        expect(actual).toEqual(expected);
+    });
+
+    it('should return 0 when called with no value', () => {
+        const product = helpers.multiplyByTen('');
+        expect(product).toBe(0);
+    });
+
+    it('should throw error when called with a string', () => {
+        expect(() => { helpers.multiplyByTen('five') }).toThrow();
+    });
+});
+
+//============ SUBTRACT BY 5
+describe('subtractFive()', () => {
+    it('should subtract provided number by five', () => {
+        const expected = 25;
+        const actual = helpers.subtractFive(30);
+        expect(actual).toEqual(expected);
+    });
+});
+
+//============ SAME LENGTH STRINGS
+describe('areSameLength()', () => {
+    it('should return true if both strings are the same length', () => {
+        const expected = true;
+        const actual = helpers.areSameLength('hello', 'howdy');
+        expect(actual).toEqual(expected);
+    });
+});
+
+//============ DEEP EQUAL!
+describe('areEqual', () => {
+    it('should return true if both variables are deep equal', () => {
+        const expected = true;
+        const x = 'Test123!';
+        const y = 'Test123!';
+        const actual = helpers.areEqual(x, y);
+        expect(actual).toEqual(expected);
+    });
+});
+
+//============ LESS THAN 90
+describe('lessThanNinety()', () => {
+    it('should return true if number is less than 90', () => {
+        const expected = true;
+        const actual = helpers.lessThanNinety(89);
+        expect(actual).toEqual(expected);
+    });
+    it('should return false if number is greater than 90', () => {
+        const expected = false;
+        const actual = helpers.lessThanNinety(91);
+        expect(actual).toEqual(expected);
```

Comment:
More specifically, the test should return false if the number is greater than or equal to 90.